### DISCUSSION TITLE
STT - nova-2 model is not returning any transcript where as with "base" model transcript is returned correctly.

### DISCUSSION THREAD

**Discussion by Nitin-390** [deepgram/community#762] [General help]
Please refer the code snipped below. Python version 3.8 and Deepgram SDK version 2.12.0

Attached audio file (https://drive.google.com/file/d/1N829lb0aCu-gysO0_4lWYyq5CMyc_fS3/view?usp=sharing)  does not return any transcript with "nova-2"model whereas it works fine without any specific model or base model

 another audio file... transcript is much better with "nova-2" but not that great with base

I believe "nova-2" is the latest model... It is difficult to understand why in 1st case nova-2 did not work. Please help here. I have tried mp3 and wav both no success with nova-2

 -----------
dg_client = Deepgram(API_KEY)
with open("audio-test.wav", 'rb') as audio:
    source = {'buffer': audio, 'mimetype': 'audio/wav'}
    response = dg_client.transcription.sync_prerecorded(source, {'model' : "nova-2","language": "en-US", 'detect_entities':False, 'detect_language':True})
    print('API response from Deepgram returned')
    print(response)

------

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by Nitin-390** [comment]
request id 8f9aa6f8-0bc3-44a6-9643-5d146c2b7f34
url = POST /v1/listen?model=nova-2&language=en-US&detect_entities=false&detect_language=true

**Comment by jkroll-deepgram** [comment]
Hi @Nitin-390, thanks for including the audio and request info. I've listened to your audio and it is pretty challenging, being only 4 seconds, low bandwidth, with static background noise. Our Nova-2 model tends to be more conservative in transcribing lower confidence audio. If this sample audio is representative of your audio type, it could be the case that Base is the best fit for your use case.

I do also notice that you're specifying `language=en-US&detect_language=true` - while it's not the cause of your issue, these are conflicting parameters - use `language=en-US` for audio you want to be transcribed in English, and `detect_language=true` for audio where you're not sure of the language and want Deepgram to detect and transcribe in the most likely language.

**Comment by Nitin-390** [comment]
Thanks @jkroll-deepgram  for your concise response! 

Problem is that I have similar audio clips where Nova-2 does better as compared to base model So it becomes difficult to understand which one to use. 

But mostly Nova-2 was behaving better than base
